apps/eightpercent/views.py

```diff
@@ -1,3 +1,53 @@
-# from django.shortcuts import render
+from rest_framework import status
+from rest_framework.generics import GenericAPIView
+from rest_framework.mixins import CreateModelMixin, ListModelMixin
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
 
-# # Create your views here.
+from apps.eightpercent.models import Account
+from apps.eightpercent.serializers import ReadAccountSerializer
+
+
+class AccountView(CreateModelMixin, ListModelMixin, GenericAPIView):
+
+    queryset = None
```

Comment:
음 저라면 queryset에 작성해줏고 함수들에서 super().get_queryset()으로 작업 할 거 같습니다.
```suggestion
    queryset = Account.objects.all()
```